grpc-examples/src/bin/greeter_client_multi.rs

```diff
@@ -0,0 +1,29 @@
+extern crate grpc_examples;
+extern crate grpc;
+extern crate futures;
+
+use grpc::Client;
+
+use grpc_examples::helloworld_grpc::*;
+use grpc_examples::helloworld::*;
+
+use std::env;
+
+
+fn main() {
+    let name = env::args().nth(1).map(|s| s.to_owned()).unwrap_or_else(|| "world".to_owned());
+
+    let client = Client::new_plain("localhost", 50051, Default::default()).unwrap();
```

Comment:
I think it would make sense to rename grpc::Client to grpc::Channel. It would align more with the terminology used in the official grpc libraries.

@stepancheg Let me know if you think we should rename grpc::Client to grpc::Channel.